src/components/Menu.js

```diff
@@ -40,20 +40,19 @@ const Menu = () => {
             <Nav.Link
               id={itemId}
               key={i + menuItem.url}
+              href={path}
               className={
                 "menu-item menu-item-type-custom menu-item-object-custom menu-item-home " +
                 itemId
               }
             >
-              <UniversalLink
-                to={path}
-                activeClassName={"current-menu-item current_page_item nav-link"}
-              >
-                {menuItem.label}
-              </UniversalLink>
+              {menuItem.label}
             </Nav.Link>
           )
         })}
+        <button href="/" className="btn gradient-btn">
```

Comment:
This should be a React Bootstrap button.

(example)
src/components/GetAnAdvantage.js

Reply:
Button updated to React Bootstrap Button